examples/SMAC4HPO_mlp.py

```diff
@@ -0,0 +1,144 @@
+"""
+An example for the usage of SMAC within Python.
+We optimize a simple MLP on the MNIST digits dataset using "Hyperband" intensification.
+"""
+
+import logging
+import warnings
+
+import numpy as np
+from sklearn.model_selection import cross_val_score
+from sklearn.neural_network import MLPClassifier
+from sklearn import metrics
+from sklearn.datasets import load_digits
+from sklearn.exceptions import ConvergenceWarning
+
+import ConfigSpace as CS
+from smac.configspace import ConfigurationSpace
+from ConfigSpace.hyperparameters import CategoricalHyperparameter, \
+    UniformFloatHyperparameter, UniformIntegerHyperparameter
+
+from smac.scenario.scenario import Scenario
+from smac.facade.smac_hpo_facade import SMAC4HPO
+from smac.intensification.hyperband import Hyperband
+
+digits = load_digits()
+
+
+# Target Algorithm
+def mlp_from_cfg(cfg, seed, instance, budget, **kwargs):
+    """
+        Creates a MLP classifier from sklearn and fits the given data on it.
+        This is the function-call we try to optimize. Chosen values are stored in
+        the configuration (cfg).
+
+        Parameters:
+        -----------
+        cfg: Configuration
+            configuration chosen by smac
+        seed: int or RandomState
+            used to initialize the rf's random generator
+        instance: str
+            used to represent the instance to use (just a placeholder for this example)
+        budget: float
+            used to set max iterations for the MLP
+
+        Returns:
+        -----------
+        np.mean(accuracy): float
```

Comment:
That's not correct, you return `accuracy * -1`. How about you return the misclassification error (`1 - accuracy`)? I'd find that far more intuitive. Also, it is sufficient to state the type of the return value here.

Reply:
Please only state the return type (and remove the two following lines).